app/src/main/java/com/example/africanid/MainActivity.java

```diff
@@ -3,12 +3,64 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Button button = findViewById(R.id.buttonS);
+
+        final TextView label1 = findViewById(R.id.textLabel1);
+        final TextView label2 = findViewById(R.id.textLabel2);
```

Comment:
Same as above here would be better if you give them a more descriptive name for those 
label1 and label2, just for keep a more understandable code.